Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -41,7 +41,7 @@ namespace AZ::Render
             );
         }, AZ::ConsoleFunctorFlags::DontReplicate,
         "Toggles debugging information display.\n"
-        "Usage: r_displayInfo [0=off/1=show/2=enhanced/3=compact]"
+        "Usage: r_displayInfo [0=off/1=show/2=enhanced/3=compact/4=fps]" // CARBONATED: changed string parameter in macro
```

Comment:
Please #if defined(CARBONATED) these lines, otherwise they're difficult to find/replace.  I know it's verbose and not necessarily neat.
#if defined(CARBONATED)
    "Toggles... new"
    "Usage... new"
#else
    "Toggles... old"
    "Usage... old"
#endif

Reply:
Problem, this string is used as a macro parameter for macro definition AZ_CVAR.
If I tried to do as you described above,
```
#if defined (CARBONATED)
        "Toggles debugging information display.\n"
        "Usage: r_displayInfo [0=off/1=show/2=enhanced/3=compact/4=fps]"
#else
        "Toggles debugging information display.\n"
        "Usage: r_displayInfo [0=off/1=show/2=enhanced/3=compact]"
#endif
```
 I see bad warning as error "warning C5101: use of preprocessor directive in function-like macro argument list is undefined behavior"
 
Ok, I will try to do something more suitable.

Reply:
Updated (as it is possible)